amida-tech/amida-survey-microservice
# Feature/ser 147

#### What's this PR do? Adds scale type questions
#### Related JIRA tickets: SER-147
#### How should this be manually tested?

Create/get/export/import scale questions and answer them.

1) Scale questions are specified using type `scale` and property `scaleLimits: {min: , max: }`. 
2) Min and max are optional but at least one of them should be present.
3) When bot present max needs to be greater or equal to min.
4) Answers to scale questions are specified using numberValue which accepts float.
5) Answers will be validated when posted according to min and max.

#### Any background context you want to provide? NA
#### Screenshots (if appropriate): NA
